tests/test_logic.py

```diff
@@ -152,6 +152,19 @@ def test_get_settings_no_files():
     assert mock_json_dump.call_count == 1
 
 
+def test_get_workspace():
+    """
+    Normally return generated folder otherwise user value
+    """
+    should_be = os.path.join(mu.logic.HOME_DIRECTORY,
+                             mu.logic.WORKSPACE_NAME)
+    assert mu.logic.get_workspace_dir() == should_be
```

Comment:
Keep in mind that as it is, executing `get_workspace_dir()` will execute `get_settings_path()` which will create the settings.json file on the default location. Even if we don't care about creating this file, if a different test has also done this (in which case it should probably be changed), then we might be reading settings data that might be different than expected.
So we should probably mock `get_settings_path` and the file open to read an empty json object.